data/network/src/commonMain/kotlin/social/androiddev/common/network/MastodonApiKtor.kt

```diff
@@ -36,29 +40,62 @@ internal class MastodonApiKtor(
         }
     }
 
+    override suspend fun createAccessToken(
+        domain: String,
+        clientId: String,
+        clientSecret: String,
+        redirectUri: String,
+        grantType: String,
+        code: String,
+        scope: String
+    ): Result<Token> {
+        return runCatchingIgnoreCancelled<Token> {
+            httpClient.post {
+                url {
+                    host = domain
+                    path("/oauth/token")
+                }
+                contentType(ContentType.Application.Json)
+                setBody(
+                    CreateAccessTokenBody(
+                        scope = scope,
+                        code = code,
+                        clientId = clientId,
+                        clientSecret = clientSecret,
+                        redirectUri = redirectUri,
+                        grantType = grantType,
+                    )
+                )
+            }.body()
+        }.onFailure { t ->
+            t.printStackTrace()
```

Comment:
my preference is to default to `throw RuntimeException("TODO add error handling + $e.cause").  this way we don't fail silently later if we  forget to add the handling

Reply:
Agreed - will resolve in a follow-up PR for here and other places as well.